libs/s25main/ingameWindows/iwWares.cpp

```diff
@@ -59,16 +63,16 @@ static void addElement(ctrlGroup& page, const glFont* font, const DrawPoint btPo
     page.AddText(600 + idOffset, txtPos, "", COLOR_YELLOW, FontStyle::RIGHT | FontStyle::BOTTOM, font);
 }
 
-// 167, 416
-iwWares::iwWares(unsigned id, const DrawPoint& pos, const Extent& size, const std::string& title, bool allow_outhousing,
-                 const glFont* font, const Inventory& inventory, const GamePlayer& player)
-    : IngameWindow(id, pos, size, title, LOADER.GetImageN("io", 5)), inventory(inventory), player(player), numPages(0)
+iwWares::iwWares(unsigned id, const DrawPoint& pos, unsigned additionalYSpace, const std::string& title,
+                 bool allow_outhousing, const glFont* font, const Inventory& inventory, const GamePlayer& player)
+    : IngameWindow(id, pos, Extent(167, 416), title, LOADER.GetImageN("io", 5)), inventory(inventory), player(player),
```

Comment:
Wasn't the (min) height 415 (415 constant and 455=415+40)? And although you are resizing later I think it would be clearer to add it here:
```suggestion
    : IngameWindow(id, pos, Extent(167, 415 + ), title, LOADER.GetImageN("io", 5)), inventory(inventory), player(player),
```
```suggestion
    : IngameWindow(id, pos, Extent(167, 415 + additionalYSpace), title, LOADER.GetImageN("io", 5)), inventory(inventory), player(player),
```

Reply:
We can do it here but then we have to precompute the number of rows needed here because the size depends on it.

Reply:
Yeah we can't reasonably do that. The idea was to show right here where `additionalYSpace` is used instead of just the magic number for the height